tests/integration/monitoring/test-monitoring.go

```diff
@@ -170,9 +167,15 @@ func testTargetsAreHealthy() {
 
 }
 
-func hasAnyLabel(target, anyOf promAPI.Labels) bool {
-	for l, _ := range target {
-		if target[l] == anyOf[l] {
+func shouldIgnoreTarget(target promAPI.Labels) bool {
+	var jobsToBeIgnored = []string{
+		// Note: These targets will be tested here: https://github.com/kyma-project/kyma/issues/6457
+		"knative-eventing/knative-eventing-event-mesh-dashboard-broker",
+		"knative-eventing/knative-eventing-event-mesh-dashboard-httpsource",
+	}
+
+	for _, j := range jobsToBeIgnored {
+		if target["job"] == j {
```

Comment:
(nit) Does it make sense to use lower case `target["job"]`?

Reply:
I'd keep it as is for now. Let's take this offline.